src/reducers/url-state.js

```diff
@@ -257,6 +257,15 @@ const invertCallstack: Reducer<boolean> = (state = false, action) => {
   }
 };
 
+const showUserTimings: Reducer<boolean> = (state = false, action) => {
```

Comment:
Please add some light documentation at the top of this function. See the functions above for examples.